### PR TITLE
print error if typing wrong template name

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

**Comment by AndyWu2015** [comment]
hey @gamerson  do we need to include this commit into blade1.0.1 repo with ide3.0.1 release ?

File: com.liferay.blade.cli/test/com/liferay/blade/cli/CreateCommandTest.java [hunk 1]
```diff
@@ -663,6 +663,19 @@ public void testListTemplates() throws Exception {
...
 
+	@Test
+	public void testWrongTemplateTyping() throws Exception {
+		String[] args = {
+			"create", "-d", "generated/test", "-t", "activatorXXX", "wrong-activator"
+		};
+
+		new bladenofail().run(args);
+
+		String projectPath = "generated/test/bar-activator";
```
Comment (gamerson) [inline com.liferay.blade.cli/test/com/liferay/blade/cli/CreateCommandTest.java]: the project path should be "generated/test/wrong-activator"

File: com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java [hunk 2]
```diff
@@ -430,28 +455,30 @@ private boolean isEmpty(String str) {
 		return false;
 	}
 
+	private boolean isExistingTempalte( String templateName ) throws Exception
```
Comment (gamerson) [inline com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java]: should be "isExistingTemplate"

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
closing see comments.

**Comment by AndyWu2015** [comment]
done

**Comment by gamerson** [comment]
Please resend, thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**